src/particleContainer/adapter/RDFCellProcessor.cpp

```diff
@@ -97,6 +118,13 @@ void RDFCellProcessor::processCellPair(ParticleCell& cell1, ParticleCell& cell2,
 
 					if (dd < _cutoffRadiusSquare) {
 	                    _rdf->observeRDF(molecule1, molecule2, dd);
+						if (_rdf->doARDF() == true) {
```

Comment:
```suggestion
						if (_rdf->doARDF()) {
```
simplify this statement